src/string_bytes.cc

```diff
@@ -62,6 +62,10 @@ class ExternString: public ResourceType {
       return length_;
     }
 
+    // Maximum length in bytes that V8 can handle. 
+    // This equals to 268435440 bytes
+    static const size_t kMaxLength = (256 * 1024 * 1024) - 16;
```

Comment:
That's right. There is no such thing in v0.12 and kStringMaxLength is not exposed anywhere.